includes/polling/applications/rrdcached.inc.php

```diff
@@ -40,13 +40,21 @@
     $sock = fsockopen($device['hostname'], 42217, $errno, $errstr, 5);
 
     if (! $sock) {
+        d_echo("\nNo Socket to rrdcached server " . $device['hostname'] . ":42217 try to get rrdcached from SNMP\n");
```

Comment:
These changes should move to its own pull request

Reply:
You're right @Jellyfrog.
I dont know how to do that, thats why it came up to this Pull... I'm not 100% confortable with git. But I'll try to do just that, so that this RRDCached CHG has its own Pull.
Thank you for your comments and advice.

Reply:
done. thx.